### PR TITLE
gcc: add pt_BR translation

### PR DESCRIPTION
**Pull request by xRuiAlves**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/common/gcc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Permitir avisos na compilação:
+
+`gcc {{arquivo_fonte.c}} -Wall -Og -o {{arquivo_executável}}`
+
+- Incluir bibliotecas de um local diferente:
+
+`gcc {{arquivo_fonte.c}} -o {{arquivo_executável}} -I{{caminho/para/header}} -L{{caminho/para/biblioteca}} -l{{nome_biblioteca}}`
+
+- Compilar o código fonte para instruções do Assembler:
```
Comment (andrik) [inline pages.pt_BR/common/gcc.md]:
```suggestion
- Compilar o código fonte para instruções Assembler:
```

File: pages.pt_BR/common/gcc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# gcc
+
+> Executa pré-processamento e compila arquivos de código fonte C e C++, efetuando também as fases de assembling e linking.
+> Mais informações: <https://gcc.gnu.org>.
+
+- Compilar múltiplos arquivos de código fonte, produzindo um arquivo executável:
+
+`gcc {{arquivo_fonte1.c}} {{arquivo_fonte2.c}} -o {{arquivo_executável}}`
+
+- Permitir avisos na compilação:
```
Comment (andrik) [inline pages.pt_BR/common/gcc.md]:
```suggestion
- Habilitar avisos durante a compilação:
```

File: pages.pt_BR/common/gcc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# gcc
+
+> Executa pré-processamento e compila arquivos de código fonte C e C++, efetuando também as fases de assembling e linking.
```
Comment (andrik) [inline pages.pt_BR/common/gcc.md]:
```suggestion
> Compilador de arquivos de código fonte C e C++, efetuando também as fases de pré-processamento, assembling e linking.
```

**Comment by sbrl** [comment]
CC @andrik

**Comment by andrik** [comment]
Hi @xRuiAlves I requested some changes, do you agree?

**Comment by sbrl** [comment]
Ah, I see. I don't speak the language, but I hadn't noticed that your comments hadn't been addressed yet @andrik

**Comment by xRuiAlves** [comment]
@andrik Implemented requested changes. Sorry for the waiting, I hadn't noticed you had made suggestions regarding this PR :stuck_out_tongue:

**Pull request merged**